src/goto-instrument/goto_instrument_parse_options.h

```diff
@@ -87,7 +88,6 @@ Author: Daniel Kroening, kroening@kroening.com
   "(show-symbol-table)(show-points-to)(show-rw-set)" \
   OPT_TIMESTAMP \
   "(show-natural-loops)(show-lexical-loops)(accelerate)(havoc-loops)" \
-  "(string-abstraction)" \
```

Comment:
@tautschnig Why has this been removed?

Reply:
Oh, my apologies for the obscure change: this is handled by `OPT_CONFIG_LIBRARY`.

Reply:
That's OK.  Now you mention it I have a nasty feeling I might have done that so... 😐